bikeshed/boilerplate/whatwg/footer.include

```diff
@@ -10,3 +10,4 @@ if ("serviceWorker" in navigator) {
   navigator.serviceWorker.register("/service-worker.js");
 }
 </script>
+<script src="https://resources.whatwg.org/standard-mdn-annos.js"></script>
```

Comment:
I think the bottom of the page is probably not the right place for this. I'd suggest at the top, but with the `defer=""` attribute so that it executes at the bottom. That way it gets fetched ASAP.

Reply:
> I think the bottom of the page is probably not the right place for this. I'd suggest at the top, but with the `defer=""` attribute so that it executes at the bottom. That way it gets fetched ASAP.

OK, thanks — moved it to the sop